### PR TITLE
App

### PR DESCRIPTION
**Pull request by Oscar-Espinoza**
This pull request updates the entire real-time-market-info-web-app project to the latest version. The update includes the following changes:

- Updated dependencies to the latest versions
- Added new features and functionality
- Fixed bugs and issues
- The project now runs on the latest version of Node.js and has been tested to ensure that all features and functionality are working as expected. All tests pass successfully and the project builds without any errors.

Please review the changes and merge this pull request once approved.

Thank you!

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Oscar-Espinoza, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

OopsNextGIF (3)

You have submitted a project that does not follow the right [read me template](https://github.com/microverseinc/readme-template). The dev branch seems to have a resemblance of the template, however, the current branch modified it.

Kindly follow the template as the previous reviewers have suggested in their comments.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Oscar-Espinoza ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] As stated in the project requirements, could you record a video presenting the features of the project and highlight a piece of the code that you found exciting or you are proud of making?

- [x] Kindly make sure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 

- [x] As stated in the project requirements, kindly make sure the project is deployed properly, the home and detail page works as expected, and the link to the deployed project is present in your README file.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

  Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/test_case/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import store from '../../redux/store';
+import Home from '../Home/Home';
+
+require('@testing-library/jest-dom');
+
+describe('Test for Home component', () => {
+  it('renders Home component without any error', () => {
+    const output = render(<Provider store={store}><Home /></Provider>);
+    expect(output).toMatchSnapshot();
+  });
```
Comment (OLIPLICHE) [inline src/components/test_case/Home.test.js:12]: # Optional:
- I would advise you to add unit tests for several components of the app and some pure functions in Redux code. It is the requirements of the projects
Capture1

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/:currencyName" element={<CurrencyDetails />} />
+        <Route path="/:currencyName/history" element={<CurrencyHistory />} />
+      </Routes>
+    </div>
```
Comment (OLIPLICHE) [inline src/App.js:15]: Your App👍 👍 👍 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Oscar-Espinoza, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- The App is working fine 👍 
- Linter is green 👍 
- Githflow is well used 👍 

 ## Required Changes ♻️
_Check the comments under the review._

# Optional

- [x] To avoid a lot of invalidation of your project, I would advise you and **strongly recommend** that you add `Getting Started` section of the Readme in your Readme Template. It is part of the general requirements, thus making your project looks more professional.
- NB: All the sections need to be filled out, do not leave an empty section


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Oscar-Espinoza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Updated Video recording 👏 👏
- Added unit Test👏 👏
- Updated Readme template 👏 👏
- The App is working fine 👏 👏
- CONGRATULATIONS👏 👏👏 👏👏 👏👏 👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**